src/grpc/admin.rs

```diff
@@ -8,6 +8,7 @@
 // licenses.
 
 use std::sync::Arc;
+use tracing::{instrument, info, error, debug};
```

Comment:
I don't know how you feel about tracing/logging, but it seems like we need to get some kind of structured something in here.  This library feels pretty fantastic for doing it.  We can pull it back out for something better, but it felt good to me to use.